client.go

```diff
@@ -339,6 +339,6 @@ func readProto(proto string, r io.Reader) error {
 	case "na":
 		return ErrNotSupported
 	default:
-		return errors.New("unrecognized response: " + tok)
+		return errors.New("unrecognized response: " + string(tok))
```

Comment:
Nit: `fmt.Errorf` would probably be nicer.

Reply:
fixed.